include/zenoh-pico/system/link/raweth.h

```diff
@@ -44,6 +45,7 @@ typedef struct {
     uint16_t vlan_type;                 // Vlan ethtype
     uint16_t tag;                       // Vlan tag
     uint16_t ethtype;                   // Ethertype of frame
```

Comment:
Do we need `ethtype` then? As far as I understand it, `data_legnth` replaces `ethtype`.

Reply:
802.3 raw requires us to use a 802.2 LLC header which would be 3 bytes of useless information for us, so I figured an ethertype and a frame length was the most efficient choice.